<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>
<body>

<h1>Applications overview</h1>

<details open>
  <summary>Transcriptomic applications</summary>
  <table>
    <tr><th>Application</th><th class="price-col">Prize</th><th class="desc-col">Analyses included</th></tr>
    <tr><td class="app-name"><a href="rnaseq/rnaseq.html" target="_blank" >Basic RNAseq (gene level)</a></td><td>157,86</td><td>Data Quality Control, Trimming, Alignment, Gene expression quantification, Differential expression profiling</td></tr>
    <tr><td class="app-name"><a href="rutas/rutas.html" target="_blank" >Basic pathway enrichment</a></td><td>140,32</td><td>Functional enrichment analysis (GSEA and/or ORA)</td></tr>
	<tr><td class="app-name"><a href="rnaseq/rnaseq.html" target="_blank">RNAseq</a> + <a href="rutas/rutas.html" target="_blank">Pathway enrichment</a></td><td>263,1</td><td>Data Quality Control, Trimming, Alignment, Gene expression quantification, Differential expression profiling, Functional GO enrichment analysis (GSEA and/or ORA)</td></tr>
    <tr><td class="app-name"><a href="splicing.html">Splicing</a></td><td>87,7</td><td>Event-based alternative splicing analysis with specialized software</td></tr>
    <tr><td class="app-name"><a href="small_rna.html"target="_blank" >Small RNA-Seq</a></td><td>192,94</td><td>Data quality control, Trimming, Alignment to reference genome, Alignment to known microRNAs, Quantification of microRNA expresion, Normalization, Differential expression analysis, Novel microRNAs discovery, Visualization of the results</td></tr>
    <tr><td class="app-name"><a href="small_rna_enrichment.html"target="_blank" >Basic pathway enrichment of small RNA-Seq</a></td><td>149,09</td><td>Functional GO enrichment analysis of miRNA-mRNA pathways</td></tr>
  </table>
</details>

<details>
  <summary>Epigenomic applications</summary>
  <table>
    <tr><th>Application</th><th class="price-col">Prize</th><th class="desc-col">Analyses included</th></tr>
    <tr><td class="app-name"><a href="chipseq.html"target="_blank" >Basic ChipSeq</a></td><td>175,4</td><td>Data Quality Control, Trimming, Alignment, Peak calling broad/narrow peaks, normalised bigWig files, gene-body meta-profile, ChIP-seq quality metrics, Peak annotation to gene features, creation of consensus peakset across all samples</td></tr>
    <tr><td class="app-name"><a href="atacseq.html"target="_blank" >Basic ATACseq</a></td><td>193,38</td><td>Data Quality Control, Trimming, Alignment, Peak calling broad/narrow peaks, normalised bigWig files, gene-body meta-profile, ChIP-seq quality metrics, Peak annotation to gene features, creation of consensus peakset across all samples</td></tr>
    <tr><td class="app-name"><a href="chipseq_dbs.html" target="_blank" >Basic ChipSeq + Differential Binding Sites (DBS) + Enrich Motifs</a></td><td>306,95</td><td>Count reads in peaks, Differential binding sites detection (DBS), motifs analysis</td></tr>
    <tr><td class="app-name"><a href="atacseq_dar.html" target="_blank" >Basic ATACseq + Differential Accesible Regions (DAR) + Enrich Motifs</a></td><td>368,48</td><td>Count reads in peaks, Differential accesible sites detection (DARs), motifs analysis</td></tr>
  </table>
</details>

<details>
  <summary>Single Cell applications</summary>
  <table>
    <tr><th>Application</th><th class="price-col">Prize</th><th class="desc-col">Analyses included</th></tr>
    <tr><td class="app-name"><a href="scrna_basic.html" target="_blank" >Basic scRNAseq</a></td><td>438,5</td><td>Data Quality Control, Trimming, Alignment, Gene expression quantification, Cell demultiplexing and unique molecular identifier (UMI)-count table generation, Cell quality control, Normalization, Feature Selection, Dimensionality reduction, Cell clustering, Marker genes identification, Cell type annotation, Differential gene expression, Differential cell type number and visualization</td></tr>
    <tr><td class="app-name"><a href="scrna_pathway.html" target="_blank" >Basic scRNAseq + pathway enrichment</a></td><td>543,74</td><td>Functional enrichment analysis (GSEA and/or ORA)</td></tr>
    <tr><td class="app-name"><a href="scrna_rnaseq_integration.html" target="_blank" >scRNAseq + RNAseq integration + pathway enrichment</a></td><td>596,36</td><td>Deconvolution, manual curation of results, Functional enrichment analysis (GSEA and/or ORA)</td></tr>
  </table>
</details>

<details>
  <summary>Variants applications</summary>
  <table>
    <tr><th>Application</th><th class="price-col">Prize</th><th class="desc-col">Analyses included</th></tr>
    <tr><td class="app-name"><a href="wgs_short.html" target="_blank" >Basic WGS (DNAseq) Short reads</a></td><td>245,56</td><td>Data Quality Control, Trimming, Alignment, Variant calling (SNP, InDels <50bp), Variant annotation effect + coverage plots</td></tr>
    <tr><td class="app-name"><a href="wgs_cnv.html" target="_blank" >Basic WGS (DNAseq) + CNV Short reads</a></td><td>350,8</td><td>Data Quality Control, Trimming, Alignment, Variant calling (SNP, InDels <50bp), Variant annotation effect + detection of CNVs → gain/loss copies fragments (longer than 50 bp) + coverage plots</td></tr>
    <tr><td class="app-name"><a href="wgs_long.html" target="_blank" >Basic WGS (DNAseq) long reads + Structural Variants + CNVs Long reads</a></td><td>456,04</td><td>Data Quality Control, Trimming, Alignment, Variant calling (SNP, InDels <50bp), Variant annotation effect + Structural Variants (SV) detection + detection of CNVs → gain/loss copies fragments (longer than 50 bp) + coverage plots</td></tr>
  </table>
</details>

<details>
  <summary>Assembly applications</summary>
  <table>
    <tr><th>Application</th><th class="price-col">Prize</th><th class="desc-col">Analyses included</th></tr>
    <tr><td class="app-name"><a href="basic_assembly.html" target="_blank" >Basic assembly</a></td><td>78,93</td><td>Automatic de novo genome assembly with specialized software</td></tr>
    <tr><td class="app-name"><a href="assembly_10kb.html" target="_blank" >Assembly < 10 kb</a></td><td>263,1</td><td>Validation of automatic genome assembly, including contig length assessment, coverage visualization, genome annotation, and evaluation of assembly and annotation completeness</td></tr>
    <tr><td class="app-name"><a href="assembly_100kb.html" target="_blank" >Assembly 10 kb-100 kb</a></td><td>280,64</td><td>“Same”</td></tr>
    <tr><td class="app-name"><a href="assembly_1mb.html" target="_blank" >Assembly 100 kb-1 Mb</a></td><td>350,8</td><td>“Same”</td></tr>
    <tr><td class="app-name"><a href="assembly_10mb.html" target="_blank" >Assembly 1-10 Mb</a></td><td>631,44</td><td>“Same”</td></tr>
    <tr><td class="app-name"><a href="assembly_50mb.html" target="_blank" >Assembly 10-50 Mb</a></td><td>920,85</td><td>“Same”</td></tr>
    <tr><td class="app-name"><a href="assembly_gt_50mb.html"target="_blank" >Assembly >50 Mb</a></td><td>A determinar</td><td>“Same”</td></tr>
  </table>
</details>

<details>
  <summary>Data submission applications</summary>
  <table>
    <tr><th>Application</th><th class="price-col">Prize</th><th class="desc-col">Analyses included</th></tr>
    <tr><td class="app-name"><a href="basic_assembly.html" target="_blank" >Raw data submission to public repositories</a></td><td>43,85</td><td>Creation of report study in repository, sample registration, uploading of raw reads and submission</td></tr>
    <tr><td class="app-name"><a href="assembly_10kb.html" target="_blank" >Submission assembly + annotation to public repositories</a></td><td>70,16</td><td>Creation of report study in repository, standarize genome annotation file, sample registration and submission</td></tr>
  </table>
</details>


<details>
  <summary>Metagenomics applications</summary>
  <table>
    <tr><th>Application</th><th class="price-col">Prize</th><th class="desc-col">Analyses included</th></tr>
    <tr><td class="app-name"><a href="raw_submission.html" target="_blank" >Raw data submission to public repositories</a></td><td>43,85</td><td>Creation of report study in repository, sample registration, uploading of raw reads and submission</td></tr>
    <tr><td class="app-name"><a href="annot_submission.html"target="_blank" >Submission assembly + annotation to public repositories</a></td><td>70,16</td><td>Creation of report study in repository, standarize genome annotation file, sample registration and submission</td></tr>
    <tr><td class="app-name"><a href="amplicons_short.html" target="_blank" >Amplicons metagenomics 16S/18S/ITS (Shortreads)</a></td><td>192,94</td><td>Data quality control, Trimming, Qiime2 (Merging reads, Deoinsing, Clustering, Taxonomic assignment, output tables generation, Diversity & statistical analysis)</td></tr>
    <tr><td class="app-name"><a href="amplicons_long.html" target="_blank" >Amplicons metagenomics 16S/18S/ITS (Longreads)</a></td><td>A determinar</td><td>Data quality control, Trimming, Error correction, Dereplication, Clustering, Denoising, Taxonomic assignment, output tables generation, Diversity & statistical analysis</td></tr>
    <tr><td class="app-name"><a href="shotgun.html" target="_blank" >Shotgun metagenomics</a></td><td>A determinar</td><td>Data quality control, Trimming, DNA host removing, de novo assembly, Taxonomic profiling (kraken, centrifuge, metaPhIAn, BLAST, DIAMOND), Gene functional annotation (Prokka, eggNOG-mapper HUMAnN3), statistical analysis (phyloseq..deseq…)</td></tr>
  </table>
</details>

</body>
</html>
